scripts/generate_provenance

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+# Takes as input a TOML file, and builds a binary and its provenance from the
+# Git HEAD, using the build config in the TOML file. To run locally, pass the
+# name of the TOML file as input. The generated provenance file will be named
```

Comment:
Maybe the provenance should be called similarly to the input file, so as not to overwrite itself each time?

Reply:
Done.